universal-login-relayer/lib/services/IdentityService.js

```diff
@@ -13,25 +16,25 @@ class IdentityService {
     this.codec = new utils.AbiCoder();
     this.hooks = hooks;
     this.provider = provider;
+    this.counterfactualIdentityService = new CounterfactualIdentityService(wallet, ensService, authorisationService, hooks, provider);
+    this.counterfactualTransactionsService = new CounterfactualTransactionsService();
```

Comment:
`counterfactualIdentityService` and `counterfactualTransactionsService` should be created in `relayer.js` and injected into `IdentityService` in constructor.